rebasehelper/output_tools/text_output_tool.py

```diff
@@ -56,59 +57,86 @@ def print_changes_patch(cls):
         patch = cls.results_store.get_changes_patch()
         if patch is not None:
             logger_report.info('\nPatch with differences between old and new version source files:')
-            logger_report.info(patch['changes_patch'])
+            logger_report.info(cls.prepend_results_dir_short('changes.patch'))
```

Comment:
Why are you discarding `patch['changes_patch']` and using hardcoded filename instead?

Reply:
I thought it's more convenient, than extracting file base name from the full path stored in `results_store`.

Reply:
Then why are you storing the value at all? It doesn't make sense.